src/unix/linux_like/linux/gnu/mod.rs

```diff
@@ -664,32 +664,50 @@ pub const NI_MAXHOST: ::socklen_t = 1025;
 
 pub const ADFS_SUPER_MAGIC: ::c_long = 0x0000adf5;
 pub const AFFS_SUPER_MAGIC: ::c_long = 0x0000adff;
+pub const AFS_SUPER_MAGIC: ::c_long = 0x5346414F;
+pub const AUTOFS_SUPER_MAGIC: ::c_long = 0x0187;
+pub const BINDERFS_SUPER_MAGIC: ::c_long = 0x6c6f6f70;
+pub const BTRFS_SUPER_MAGIC: ::c_long = 0x9123683E;
```

Comment:
```suggestion
pub const BTRFS_SUPER_MAGIC: ::c_long = 0x9123683e;
```

Same here.

Reply:
This one isn't resolved. Also it'd be great if you could squash commits to clean up history.

Reply:
What do you mean it isn't resolved?  The CI checks passed.  Do you mean that it couldn't be resolve on your own dev machine?  Are your kernel headers too old?

Reply:
Hm? The lowercase style issue as I mentioned is unresolved here.

Reply:
Oh, I see what you mean.  I thought you meant that a symbol was unresolved by the compiler.

Reply:
Ok, I squashed all of the commits except the first.  I think the file's history is more understandable when purely stylistic changes like that one are separated from content changes.

Reply:
Ah, sorry for confusing! I'll merge this once CI passes, thanks <3